.gitignore

```diff
@@ -1,2 +1,3 @@
 Gemfile*
+up.sh
```

Comment:
If you have custom helper scripts that you use, add them to your environment/path, not into the repo.